tutorials/src/bin/basic-tutorial-4.rs

```diff
@@ -0,0 +1,109 @@
+extern crate gstreamer as gst;
+use std::io;
+use std::io::Write;
+use gst::prelude::*;
+use gst::MessageView;
+
+fn main() {
+    // Initialize GStreamer
+    gst::init().unwrap();
+
+    // Creat the playbin element
+    let playbin = gst::ElementFactory::make("playbin", "playbin")
+        .expect("Failed to create playbin element");
+
+    // Set the URI to play
+    let uri = "https://www.freedesktop.org/software/gstreamer-sdk/data/media/sintel_trailer-480p.webm";
+    playbin.set_property("uri", &uri)
+        .expect("Can't set uri property on playbin");
+
+    // Start playing
+    let ret = playbin.set_state(gst::State::Playing);
+    assert_ne!(ret, gst::StateChangeReturn::Failure);
+
+    // Listen to the bus
+    let bus = playbin.get_bus().unwrap();
+    let mut is_playing = false;
+    let mut seek_enabled = false;
+    let mut seek_done = false;
```

Comment:
Maybe add some comments what these 3 variables *mean*.

There also must be a nicer way for doing this than having 3 mutable boolean flags, but I also currently don't see an easy way for that. By using futures this might be nicer, you could wait for playing, then do the query and schedule 10s in the future the seek.

But as this should be similar to the tutorial in C (so that the text can be reused), it should be fine. If you want you can submit a second version that is nicer though :)